### ISSUE TITLE
Re-run service cannot get the scenarioLineNumber on afterScenario Hook for Cucumber

### ISSUE THREAD

**Issue by Gomezjh** [webdriverio-community/wdio-rerun-service#18]
**Environment (please complete the following information):**
 - **Node.js version:** [14.16]
 - **NPM version:** [6.14.11]
 - **webdriver.io version:** [7.5.3]
 - **@wdio/cucumber-framework version:** [7.5.3]
 - **wdio-rerun-service version:** [1.7.0"]

**Config of webdriver.io**
 
    cucumberOpts: {
        backtrace: false,
        requireModule: ['@babel/register'],
        failAmbiguousDefinitions: true,
        failFast: false,
        ignoreUndefinedDefinitions: false,
        names: [],
        snippets: true,
        source: true,
        profile: [],
        require: [
            './tests/steps/*.js',
        ],
        scenarioLevelReporter: false,
        order: 'defined',
        snippetSyntax: undefined,
        strict: true,
        // <string> (expression) only execute the features or scenarios with
        // tags matching the expression, see
        // https://docs.cucumber.io/tag-expressions/
        tagExpression: 'not @Pending',
        // <boolean> add cucumber tags to feature or scenario name
        tagsInTitle: false,
        timeout: 720000,
    },

    // ====================
    // Appium Configuration
    // ====================
    services: [
        [
            'appium',
            {
                args: {
                    relaxedSecurity: true,
                },
                command: 'appium',
                logPath: './'
            },
        ],
        [
            RerunService, 
            {
                rerunDataDir: './custom-rerun-directory',
                rerunScriptPath: './custom-path-for-rerun.sh',
            }
        ],

    ],

**Describe the bug**
When Re-run service is configured, the scenarioLineNumber cannot be obtained in the afterScenario hook, so that, the JSON data cannot be created and Re-run does not work properly 

**To Reproduce**
1. Configure Re-run service as above 
2. Run any .feature file getting one or more scenarios failed
3. See how Re-run is not performed for failed scenarios 

**Expected behavior**
Failed scenarios should be re-run

**Snapshots**

**Additional context**
I have uncomment all "console.log" on the index.js of "wdio-rerun-service" only for debug purpose, and this is the output : 

```[0-0] Configuring PREPRODUCTION environment
[0-0] RUNNING in Android - /tests/features/login.feature
[0-0] Re-run service is activated. Data directory: ./custom-rerun-directory
[0-0] Error in "1: I expect that element "Next_Bun" from "OnboardingScreen" is displayed"
Error: selector needs to be typeof `string` or `function`
    at $ (/Users/-------/Documents/Repos/------/node_modules/@wdio/runner/build/index.js:189:42)
    at OnboardingScreen.getElement (/Users/---/Documents/Repos/---/tests/screenobjects/onboarding.screen.js:26:16)
    at World.<anonymous> (/Users/---/Documents/Repos/----/tests/steps/then.js:14:38)
[0-0] Re-run service did not detect any non-passing scenarios or tests.
[0-0] FAILED in Android - /tests/features/login.feature```

**Comment by mikesalvia** [comment]
@Gomezjh Thank you for your bug, it will be groomed and planned in our sync on July 13th, 2021, we will have and update then.

**Comment by mikesalvia** [comment]
Can you confirm try with version 1.7.2?

**Comment by mikesalvia** [comment]
@Gomezjh Also can you confirm that you are using the latest `wdio-rerun-service` with the appropriate `webdriverio` version?

**Comment by Gomezjh** [comment]
Hi! @mikesalvia, in the environment part of the issue, you can find the versions, those were the latest versions on the day the issue was opened.

![image](https://user-images.githubusercontent.com/49615026/128497743-45964941-f598-4205-bbc9-2810f93f762a.png)

**Comment by mikesalvia** [comment]
@Gomezjh Thanks sorry, I just wanted to be sure since we would see this issue when using v6 compatible re-run service with v7 WDIO. I will look into this more deeply.

**Comment by mikesalvia** [comment]
@Gomezjh There is an active PR that will be merged in shortly which should resolve this issue.

**Comment by mikesalvia** [comment]
Fixed in #31 will be included in 1.7.3.
